### ISSUE TITLE
Discover: Make it easier to follow sites featured in the Discover stream

### ISSUE THREAD

**Issue by blowery** [Automattic/wp-calypso#2751]
Right now it's pretty hard to follow sites featured on Discover. The follow button is hidden behind a menu, and it's not clear if it means I would follow Discover or the site on the card. 

Consider adding another Follow button to cards that feature a site and have it near the attribution line on the card.

Or maybe make the Follow button more verbose and show it like we do on tag streams, on the right side of the header. Instead of "Follow" maybe "Follow $siteName"?

cc @kjellr @shaunandrews

**Comment by shaunandrews** [comment]
I don't think we need to do anything special for Discover posts — the follow button will still be shown in context, right next to the site component showing the original site.

**Comment by blowery** [comment]
> I don't think we need to do anything special for Discover posts — the follow button will still be shown in context, right next to the site component showing the original site.

I don't think it is right now?

![screenshot](https://cldup.com/J4lVwNghGY.png)

![screenshot](https://cldup.com/RLbo3nDQtA.png)

**Comment by blowery** [comment]
@shaunandrews ah i see, you meant in the new full post view treatment. i just landed that, so let's let it bake and see how it does.

**Comment by blowery** [comment]
Closing for now
